### PR TITLE
feat: cards added

### PR DESCRIPTION
**Pull request by deekshithmd**
completed card component

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.discount-percentage{
+    font-size: 1.3rem;
+    font-weight: 500;
+    margin-left: 0.3rem;
+    color: var(--primary-color);
+}
+
+.card-details button{
```
Comment (ashishkatiyar2404) [inline components/card/card.css:89]: using tag style is ok. But try to always use classes.

**Review by ashishkatiyar2404 — APPROVED** [review]

**Comment by ashishkatiyar2404** [comment]
Your code looks good to me, easy to read, and understandable.💯

File: components/card/card.css (hunk 1 above)
Comment (deekshithmd) [reply components/card/card.css:89]: Ok, thank you for the review

**Comment by deekshithmd** [comment]
> Your code looks good to me, easy to read, and understandable.💯

Thank you

**Review by j24m — APPROVED** [review]
Wow, I love to review when people write clean code, great work, good to merge:)

**Review by m26unkwn — APPROVED** [review]
Smooth

**Comment by deekshithmd** [comment]
> Wow, I love to review when people write clean code, great work, good to merge:)

Thank for the review

**Comment by deekshithmd** [comment]
> Smooth

Thank you for the review

**Pull request merged**